data/genomics/homo_sapiens/illumina/flowcell/SampleInfo.csv

```diff
@@ -1,6 +1,6 @@
 samplename,library,organism,tag
 Sample1,test,Homo sapiens,WES
-Sample2,test,Danio Rerio,WES
```

Comment:
This shouldn't matter, the workflow can catch variations in notation

Reply:
Okay I'll expand the `enum` for that in the schema then :)

Reply:
Yeah a regex should work, an enum of regexes is not possible as far as I know, but you can just use `anyOf` as a sort of replacement for `enum` to make it work :)